pkg/payload/preflight/features/features.go

```diff
@@ -0,0 +1,64 @@
+package features
```

Comment:
So the goal is to prevent upgrades when features are set.. 

the precondition `upgradeable == false` preventing upgrades _personally_ I think needs more thought and testing to make sure the operators are no missusing before we add is important. So I don't think we should be doing it.

KAO already setting a `uprageable = false` on features is great, but I think depending on that wouldn't be ideal as mentioned above, and also depends on KAO always respecting that :P I think it's better if CVO does it.

Reply:
^^ ping @smarterclayton

Reply:
Most of our components ALREADY report Upgradeable false